packages/server/api/models/recipe.js

```diff
@@ -27,7 +27,7 @@ const unitEnums = [
 const schema = new Schema(
   {
     servings: {
-      type: Number,
+      type: String,
```

Comment:
why is this a string now?

Reply:
We figured we would make it a string to handle cases where they say "Three servings", also we encountered ranges "2-4 servings", etc.

Reply:
hmm, I'd rather parse this into a number that we can actually use. As a string this seems a lot less useful

Reply:
I think we should probably make this another ticket if we want to have it as a number, because it would be a lot of work and the only real benefit that I can see right now is filter capabilities

Reply:
i'm on board with that